packages/components/input/__tests__/input.test.tsx

```diff
@@ -116,4 +117,32 @@ describe("Input", () => {
 
     expect(ref.current?.value)?.toBe(value);
   });
+
+  it("should clear the value and onClear is triggered", async () => {
+    const onClear = jest.fn();
+
+    const ref = React.createRef<HTMLInputElement>();
+
+    const {container} = render(
+      <Input
+        ref={ref}
+        isClearable
+        defaultValue="junior@nextui.org"
+        label="test input"
+        onClear={onClear}
+      />,
+    );
+
+    const clearButton = container.querySelector("[data-slot='clear-button']") as HTMLElement;
```

Comment:
It seems better to use `getByRole` instead of `querySelector`, considering the [priority of queries].(https://testing-library.com/docs/queries/about/#priority)

```suggestion
    const {getByRole} = render(
      <Input
        ref={ref}
        isClearable
        defaultValue="junior@nextui.org"
        label="test input"
        onClear={onClear}
      />,
    );

    const clearButton = getByRole("button");
```